models/intermediate/int_recharge__customer_details.sql

```diff
@@ -20,7 +20,6 @@ with customers as (
         round(cast(sum(total_discounts) as {{ dbt.type_numeric() }}), 2) as total_amount_discounted,
```

Comment:
See the small changes in this file. The changes are only style focused. Would you be able to make these changes to this file and also use the same methodology and apply the style standards to the other models.